README.md

```diff
@@ -38,6 +38,8 @@ Orchestrator relies on these other IoT parts:
 - Context Broker (Orion)
 - CEP (Perseo)
 - LDAP (Openldap - optional)
+- MAILER (optional)
+- MongoDB
```

Comment:
Just to confirm... MongoDB endpoint is not optional, right?

Reply:
Clarified  in 857545c04e5aed69596cd42b3b6d6f2df9ccd67a